vtkext/private/module/vtkF3DRenderer.h

```diff
@@ -652,6 +672,7 @@ class vtkF3DRenderer : public vtkOpenGLRenderer
   int RaytracingSamples = 0;
   double UpVector[3] = { 0.0, 1.0, 0.0 };
   double RightVector[3] = { 1.0, 0.0, 0.0 };
+  double UpDirection[3] = { 0.0, 1.0, 0.0 };
```

Comment:
Sorry, you were right, PendingUpVector is a better name, it should be reflected in the Setter name though